plugins/chat/test/javascripts/components/chat-user-info-test.js

```diff
@@ -0,0 +1,25 @@
+import { setupRenderingTest } from "discourse/tests/helpers/component-test";
+import { exists } from "discourse/tests/helpers/qunit-helpers";
+import hbs from "htmlbars-inline-precompile";
+import { module, test } from "qunit";
+import { render } from "@ember/test-helpers";
+
+module("Discourse Chat | Component | chat-user-info", function (hooks) {
+  setupRenderingTest(hooks);
+
+  test("avatar and name", async function (assert) {
+    this.set("user", this.currentUser);
+
+    await render(hbs`<ChatUserInfo @user={{this.user}} />`);
+
+    assert.true(
+      exists(`a[data-user-card=${this.user.username}] div.chat-user-avatar`)
+    );
+
+    assert.true(
+      exists(
+        `a[data-user-card=${this.user.username}] span.chat-user-display-name`
+      )
+    );
```

Comment:
All good, but try to use: https://github.com/mainmatter/qunit-dom/blob/master/API.md

This is what we are going to use more and more.